app/src/main/java/com/google/droidjump/GameFragment.java

```diff
@@ -33,9 +39,19 @@
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Point screen = new Point();
-        Objects.requireNonNull(getActivity()).getWindowManager().getDefaultDisplay().getSize(
-                screen);
-        gameView = new GameView(getActivity(), screen.x, screen.y, /* isPlaying= */ true);
+        Activity activity = getActivity();
+        WindowManager windowManager = Objects.requireNonNull(activity).getWindowManager();
+        Display defaultDisplay = windowManager.getDefaultDisplay();
+
+        // Writing size to the screen variable
+        defaultDisplay.getSize(screen);
+
+        // Extracting screen coordinates
```

Comment:
nit: don't really see a point in extracting coordinates like that. You use them one time and "screen.x" is descriptive enough to use directly.